### PR TITLE
Added close_screen.dart to remove redundant AlertDialog Wiget of _onBackPressed method and Resolved Screen rotation

### PR DESCRIPTION
**Pull request by pranjalg13**
Removed the redundancy of the _onBackPressed. Added a constants folder for easier changes of constant parts of the file. #6

### CODE REVIEW

**Comment by NishealJ** [comment]
Thankyou for the PR @pranjalg13 I'll test this out.

File: .gitignore [hunk 1]
```diff
@@ -35,3 +35,5 @@ lib/generated_plugin_registrant.dart
 
 # Exceptions to above rules.
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
+
+pubspeec.lock
```
Comment (NishealJ) [inline .gitignore]: Let's not keep lock files in gitignore, we need to track library updates.

File: lib/constants/close_screen.dart [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          'Yes',
+          style: TextStyle(
+            color: Colors.lightBlue,
+            fontSize: 15,
+            fontFamily: 'NotoSansJP',
+            fontWeight: FontWeight.w400,
+          ),
+        ),
+        onPressed: () {
+          //SystemNavigator.pop();
```
Comment (NishealJ) [inline lib/constants/close_screen.dart]: this is also a redundant code 😅😂

**Review by NishealJ — COMMENTED** [review]
@pranjalg13 ptal!

File: .gitignore (hunk 1 above)
Comment (pranjalg13) [reply .gitignore]: So should I include it in the commit??

File: lib/constants/close_screen.dart (hunk 2 above)
Comment (pranjalg13) [reply lib/constants/close_screen.dart]: Somewhat less redundant. 😂 But I will reduce it in following commits.

File: .gitignore (hunk 1 above)
Comment (NishealJ) [reply .gitignore]: yes please push the lock files also, we'll try to vc it later.

Comment (pranjalg13) [reply .gitignore]: Yep, I have pushed the lock file. Check it out

**Comment by pranjalg13** [comment]
Added the commits please see into it. Please also leave feedback if any changes required

**Comment by pranjalg13** [comment]
Hey, @NishealJ resolved the screen rotation issue. Please check the PR.

**Comment by NishealJ** [comment]
@pranjalg13 Please resolve merge conflicts here.

**Comment by pranjalg13** [comment]
Hey Resolved the conflicts, Please check once @NishealJ 

> @pranjalg13 Please resolve merge conflicts here.

**Comment by NishealJ** [comment]
Thanks @pranjalg13

**Pull request merged**